tests/units/classes/Project/qgisProjectForTests.php

```diff
@@ -4,8 +4,11 @@
 
 class qgisProjectForTests extends QgisProject
 {
-    public function __construct()
+    public function __construct($original = false, $data = array())
     {
+        if ($original) {
+            parent::__construct(__FILE__, new lizmapServices(null, null, false, '', ''), $data);
```

Comment:
why `__FILE__` ? the expected parameter is the name of a qgis file, not the filename of the class. Use a fake name like 'fake.qgs', it is less disturbing.

Instead of having this `$original` parameter, just have `$data = null`, and if `$data` is not null, call the parent constructor